inc/admin/dashboard/main.php

```diff
@@ -508,4 +516,22 @@ private function should_show_feedback_notice() {
 		}
 		return false;
 	}
+
+	/**
+	 * Check if the Neve Pro license is valid.
+	 *
+	 * @return bool 
+	 */
+	private function is_valid_license() {
```

Comment:
@UVLabs let's use the trait that @abaicus made here https://github.com/Codeinwp/neve/pull/3285/files#diff-24feaa43da5d874bec2db460beed2f0e3b856a14a3269282c8a5d71e8831cbecR19